### ISSUE TITLE
About the training stage

### ISSUE THREAD

**Issue by GilgameshD** [ckycky3/FHVAE-pytorch#1]
Hi, thanks for the reproduction. I am curious why there is no "optimizer.zero_grad()" in the training loop. Won't that cause the accumulation of gradient?
